src/Views/Home.css

```diff
@@ -1,18 +1,14 @@
 .Home-content {
-  padding: 0px 15px 20px 15px;
+  padding: 0px 15px 20px 25px;
 }
 
 .Home-content a {
-  color: inherit;
-  text-decoration: none;
-}
-
-.Home-content a:hover {
-  color: #888;
+  color: #0000ff;
+  text-decoration: underline;
 }
 
 .Home-content ul {
```

Comment:
You can probably use default styling if it looks OK. This was for the first navigation before bootstrap.